### DISCUSSION TITLE
"Resolving the issue of "Your GitHub Copilot subscription has ended."

### DISCUSSION THREAD

**Discussion by BalteMD** [community/community#112587] [GitHub Education] [answered]
### Hi everyone,

![image](https://github.com/community/community/assets/92046606/1fed4832-db44-44dd-b1af-3e2cdb163064)
![image](https://github.com/community/community/assets/92046606/6033a2db-8cb7-4017-ae1b-23d001d32bb7)

I successfully registered for GitHub Copilot and received an email from GitHub. After 3 days, when I logged back into GitHub, I received the error "Your GitHub Copilot subscription has ended." Has anyone encountered this issue before and can help me resolve it?

**Comment by samus-aran** [comment]
Hi @dunghm19  👋

Have you tried re-enabling Copilot? To enable Copilot, go to your account settings and under Code, planning, and automation section, select Copilot. You'll see "You're eligible to use copilot for free" and a button to claim the offer.

Let us know what you see when you try and re-enable Copilot using the instructions above. 👍

**Reply by BalteMD** [reply]
![image](https://github.com/community/community/assets/92046606/23ff0ae1-2795-4a45-bb95-8f9daa47eab4)
![image](https://github.com/community/community/assets/92046606/9b3c9f63-ab46-4670-aa96-24f9f7921b2a)

I have enabled it, everything is still the same.

**Reply by samus-aran** [reply, marked as answer]
Thanks for the screenshots! 

We can confirm you have a verified GitHub Education student coupon so nothing is obvious we can see. We do have a [GitHub Copilot FAQs + Troubleshooting common issues ⚙️](https://github.com/orgs/community/discussions/47318), perhaps the question is there else directions to reach out to support.

Let us know how you get on 👍
